datarobot_batch_scoring/main.py

```diff
@@ -106,28 +120,28 @@ def parse_args(argv, standalone=False):
 
     conn_gr = parser.add_argument_group('Connection control')
     conn_gr.add_argument('--timeout', type=int,
-                         default=None,
+                         default=defaults['timeout'],
                          help='The timeout for each post request. '
                          '(default: %(default)r)')
     conn_gr.add_argument('--n_samples', type=int,
                          nargs='?',
-                         default=False,
+                         default=defaults['n_samples'],
                          help='Specifies the number of samples '
                               '(rows) to use per batch. If not defined the '
                               '"auto_sample" option will be used.')
     conn_gr.add_argument('--n_concurrent', type=int,
                          nargs='?',
-                         default=4,
+                         default=defaults['n_concurrent'],
                          help='Specifies the number of concurrent requests '
                          'to submit. (default: %(default)r)')
     conn_gr.add_argument('--n_retry', type=int,
-                         default=3,
+                         default=defaults['n_retry'],
                          help='Specifies the number of times DataRobot '
                          'will retry if a request fails. '
                          'A value of -1 specifies an infinite '
                          'number of retries. (default: %(default)r)')
-    conn_gr.add_argument('--resume', action='store_true',
-                         default=False,
+    conn_gr.add_argument('--resume', const=True, type=str2bool,
```

Comment:
If someone was using `--resume` before, will this change break their workflow?

If so, then we have to go for the `--no-resume` route to preserve functionality.